kore/test/Test/Kore/Step/Axiom/EvaluationStrategy.hs

```diff
@@ -79,13 +94,8 @@ test_definitionEvaluation =
                             }
                     , remainders =
                         OrPattern.fromPatterns
-                        $ map (fmap mkEvaluated)
-                            [ Conditional
-                                { term = Mock.functionalConstr10 Mock.a
-                                , predicate = makeNotPredicate requirement
-                                , substitution = mempty
-                                }
-                            ]
+                        $ map (fmap mkEvaluated . introduceDefinedness ceil)
```

Comment:
This is just a duplication of the code in the tests. Please state the arguments-defined conditions explicitly in the expected test result. For most tests, there should not be an additional condition.